app/products/calls/components/calls_custom_message/calls_custom_message.tsx

```diff
@@ -123,19 +124,28 @@ export const CallsCustomMessage = ({
                     style={style.phoneHangupIcon}
                 />
                 <View style={style.messageText}>
-                    <Text style={style.startedText}>{'Call ended'}</Text>
+                    <Text style={style.startedText}>
+                        {intl.formatMessage({
+                            id: 'mobile.calls_call_ended',
+                            defaultMessage: 'Call ended',
+                        })}
```

Comment:
You can use the component FormattedText instead, I think is cleaner

Reply:
Ah, thanks for pointing that out. It is cleaner, thanks.